### ISSUE TITLE
InventoryManager default constructor

### ISSUE THREAD

**Issue by jorbor** [MadTigger/Valant#1]
What's the purpose of this? Is it only for testing?

**Comment by MadTigger** [comment]
Yep. Passing in null causes the ctor to initialize an empty dictionary, which is handy for tests. It could be omitted and just call the normal ctor with a null param.
